test/suites/edgexfoundry/config_test.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	"edgex-snap-testing/test/utils"
+	"testing"
+)
+
+const newPort = "56789"
+
+// Deprecated
+func TestEnvConfig(t *testing.T) {
+	const envServicePort = "env.device-virtual.service.port"
```

Comment:
Move the constants to the scope that they are used. In this case, the subtest. Otherwise the names are confusing out of the context(newPort of what? envServicePort of what service?).

Reply:
By moving the constants, I need to move `snapUnset` from cleanup to subtest, is this ok? Please see https://github.com/canonical/edgex-snap-testing/pull/36/commits/a49b7a7e353a6ea7c5fb257420388a3a2b0fb90c